scripts/infra.sh

```diff
@@ -0,0 +1,20 @@
+if [ "$1" != "" ]; then
+    CONFIG=$(echo "$1" | tr '[:upper:]' '[:lower:]')
+    cp ~/nethermind_$CONFIG/keystore/node.key.plain $CONFIG.key
+    rm -r ~/nethermind_$CONFIG
+    cp -r ~/nethermind ~/nethermind_$CONFIG
+    cd ~/nethermind_$CONFIG
+    cp ~/NLog.$CONFIG.config ~/nethermind_$CONFIG/NLog.config
+    mkdir ~/nethermind_$CONFIG/keystore
+    cp ~/$CONFIG.key ~/nethermind_$CONFIG/keystore/node.key.plain
+    dotnet Nethermind.Runner.dll --config ../$CONFIG.cfg
+else
+    cp ~/nethermind_mainnet/keystore/node.key.plain mainnet.key
```

Comment:
in such case you should probably apply DRY principle and just assign mainnet to CONFIG when null instead of copying pasting everything